kani-driver/src/cbmc_output_parser.rs

```diff
@@ -3,7 +3,7 @@
 
 //! Module for parsing CBMC's JSON output. In general, this output follows
 //! the structure:
-//! ```
+//! ```text
```

Comment:
json?

Reply:
Not valid json.

Only important thing here is making rustdoc/rust-analyzer not thing this is a rust doctest

Reply:
fair